src/indexers/hosted/hosted-indexers.test.ts

```diff
@@ -21,13 +21,13 @@ beforeEach(async () => {
 
   const catalogName = `catalog-${Date.now()}`;
   catalog = await testClient.configureCatalog(catalogName, config);
-}, 20000);
+}, 30000);
 
 afterEach(async () => {
   if (catalog) {
     await catalog.delete(); // this will also delete any indexer referencing this catalog
   }
-}, 20000);
```

Comment:
Why does it take 30s to delete a catalog?

Reply:
Deleting a catalog is a multi-step process: you need to delete docs from TP namespace, from DB, enumerate and delete indexers, finally delete the catalog from DB. It's of course scary that it takes that long in production, but since it's a delete path, I don't think we should worry that much for now?